tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This is the smallest tsconfig possible for now (from https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html). Only addition is `strict`.